.eslintrc.yml

```diff
@@ -4,3 +4,7 @@ globals:
   shallow: readonly
   render: 'readonly'
   mount: 'readonly'
+rules:
+  sort-imports:
+    - error
+    - ignoreDeclarationSort: true
```

Comment:
Any specific reason for ignoring declaration sort? I am fine with either way, just curious.

Reply:
@karelhala I was planning to make incremental changes - Since we plan to apply the same rules to all repos., I thought I'd tread carefully.